kubernetes-go-configmap-rollout/README.md

```diff
@@ -0,0 +1,151 @@
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
+
+# App Rollout via ConfigMap Data Change
+
+Uses nginx to reverse-proxy traffic to `pulumi.github.io`. The nginx configuration is contained in
+the file `default.conf` in this directory; this program reads that file and puts it in a
+`ConfigMap`. Hence, changing data in that file will cause register as a change in the `ConfigMap`'s
+data, which will trigger a rollout of the nginx `Deployment`.
+
+![configmapRollout](images/rollout.gif "ConfigMap-induced Rollout")
+
+## Running the App
+
+Follow the steps in [Pulumi Installation and Setup](https://www.pulumi.com/docs/get-started/install/) and
+[Configuring Pulumi Kubernetes](https://www.pulumi.com/docs/intro/cloud-providers/kubernetes/setup/) to
+get setup with Pulumi and Kubernetes.
+
+Install dependencies:
+
+```sh
+go get ./...
+```
+
+Create a new stack:
+
+```sh
+$ pulumi stack init
+Enter a stack name: configmap-rollout-dev
+```
+
+This example will attempt to expose the `nginx` deployment to the Internet with
+a `Service` of type `LoadBalancer`. Since minikube does not support
+`LoadBalancer`, the application already knows to use type `ClusterIP` instead;
+all you need to do is to tell it whether you're deploying to minikube:
+
+```sh
+pulumi config set kubernetes-go-configmap-rollout:isMinikube <value>
```

Comment:
not sure if you know but you don't need to add the project name as part of the config variable:

```
pulumi config set isMinikube <value>
```

Will auto prefix it